journald/lib_journald.cpp

```diff
@@ -225,14 +251,16 @@ class JournaldContainerLoggerProcess :
     // level containers) , and the executor is named, use that name to
     // present the logs. Otherwise, default to the ExecutorID.
     // This is the value that shows up in the typical journald view.
+    label.set_key("SYSLOG_IDENTIFIER");
     if (executorInfo.isSome()) {
-      label.set_key("SYSLOG_IDENTIFIER");
       label.set_value(
           executorInfo->has_name() ?
           executorInfo->name() :
           executorInfo->executor_id().value());
-      labels.add_labels()->CopyFrom(label);
+    } else {
+      label.set_value("mesos-container");
     }
+    labels.add_labels()->CopyFrom(label);
```

Comment:
These are additional changes to those documented in the commit description. Let's make that more transparent in e.g. a ticket and/or in the commit description.

Reply:
Done.